src/app/store/reducers/Action.Constants.tsx

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+import { IUser } from "../../../features/users/types";
+import { ITask } from "../../../features/tasks/types";
+
+export const getUsersResponse = () => axios.get<IUser[]>(
+  "https://jsonplaceholder.typicode.com/users"
+);
+
+export const getTasksResponse = () => axios.get<ITask[]>(
+        "https://jsonplaceholder.typicode.com/todos"
+      );
+
```

Comment:
it's better to move a such functions to **api** file.
constants we use text/numbers/arrays/object (all static not collable values)